source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -45,6 +59,10 @@ bool BioXASSideAppController::setupDataFolder()
 void BioXASSideAppController::initializeBeamline()
 {
 	BioXASSideBeamline::bioXAS();
+	connect( BioXASSideBeamline::bioXAS()->ge32ElementDetector(), SIGNAL(connected(bool)), this, SLOT(onGeDetectorConnectedChanged()) );
+	onGeDetectorConnectedChanged();
```

Comment:
It doesn't cause a crash, because there is no entry for the detector in the component-view mapping until the view is created.